docs/tutorial/electron-versioning.md

```diff
@@ -136,12 +136,12 @@ We reconcile flagged code with our versioning strategy as follows:
 
 We seek to increase clarity at all levels of the update and releases process. Starting with `2.0.0` we will require pull requests adhere to the [Conventional Commits](https://conventionalcommits.org/) spec, which can be summarized as follows:
 
-* Commits that would result in a semver **major** bump must start with `BREAKING CHANGE:`.
+* Commits that would result in a semver **major** bump must start their body with `BREAKING CHANGE:`.
```

Comment:
Just curious: what is commit's body? Commit message?

Reply:
@tonyganch I think that normally the commit `message` is the first line of the commit text (normally less than 80 chars) and the `body` is everything after that.

```markdown
This right here is the commit message

Here you would add more details in the body

* Maybe
* Some 
* Bullet points

And so on
```

Reply:
@MarshallOfSound I've already thought that the first line is a "message's summary/subject" and the rest (after an empty line) is a "message's body".

https://chris.beams.io/posts/git-commit/